mep/people/templates/people/member_cardlist.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static mep_tags %}
+
+{% block page-subtitle %}{{ member.firstname_last }} Cards · Library Members · {% endblock %}
+
+{% block js %}
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.firstname_last style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="sections">
+    <li class="tab">
+        <a href="{% url 'people:member-detail' slug=member.slug %}">biography</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'people:membership-activities' slug=member.slug %}">activities</a>
+    </li>
+    <li class="tab" aria-selected="true">
```

Comment:
note to self to check if `aria-selected` actually makes sense or has any effect here. [w3c spec](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-selected) seems to say that it might not, since `<li>` isn't in the list of roles that use it...i seem to recall this being an interesting conundrum because we have a UI that *looks* like tabs, but is actually a navlist of links (when you click one you don't see a tab pane, you go to another page). not sure how this interacts with "the active page shouldn't be a link" either.